index.html

```diff
@@ -129,7 +130,7 @@ <h3>
                         <div class="device-box-header">Windows Mixed Reality headsets</div>
                         <span>
                             <a href="https://docs.microsoft.com/en-us/microsoft-edge/webvr/webvr-with-edge/">Microsoft Edge supports WebVR</a>. <a href="https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webvr/">Status: "The WebVR 1.1 API is fully supported."</a>.
-                            <a href="https://www.supermedium.com">Supermedium</a> also supports through SteamVR.
+                            <a href="https://webvr.rocks/firefox">Firefox</a> and <a href="https://www.supermedium.com">Supermedium</a> also support through SteamVR.
```

Comment:
can you add this link for [SteamVR](https://docs.microsoft.com/en-us/windows/mixed-reality/enthusiast-guide/using-steamvr-with-windows-mixed-reality)?

Reply:
`also support through` -> `are also supported with`?

Reply:
Sure, done.